bin/templates/project/app/build.gradle

```diff
@@ -22,33 +22,32 @@ apply plugin: 'com.android.application'
 buildscript {
     repositories {
         mavenCentral()
-        maven {
-            url "https://maven.google.com"
-        }
+        google()
         jcenter()
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.0'
+        classpath 'com.android.tools.build:gradle:3.2.0'
     }
 }
 
 // Allow plugins to declare Maven dependencies via build-extras.gradle.
 allprojects {
     repositories {
-        mavenCentral();
+        mavenCentral()
         jcenter()
     }
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '4.4.0'
+    gradleVersion = '4.6.0'
```

Comment:
Any reason for not using the latest 4.10.2? It should deliver better build speeds and other benefits.

Reply:
Historically we've targeted the minimum required version of gradle

Reply:
Is there some specific reason for targeting the minimum version? I don't think that targeting older versions has benefits over targeting newer versions. Also, Gradle development between 4.6 and 4.10 greatly focused on speed

Reply:
Also, as I mentioned in https://github.com/apache/cordova-android/issues/596 - Since Gradle 4.9 it is possible to build App Bundle - a better way to publish apps to the Play Store.
Currently, Cordova doesn't support this and I have to manually change the app manifests after each build.